src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/ReviewTimeExtensionsHandler.java

```diff
@@ -50,11 +50,14 @@ public boolean canHandle(
         AsylumCase asylumCase = callback.getCaseDetails().getCaseData();
         String decisionOutcomeDueDate = getTimeExtensionDueDate(asylumCase);
 
-        Optional<List<IdValue<Direction>>> maybeDirections = asylumCase.read(DIRECTIONS);
-        Optional<IdValue<Direction>> directionBeingUpdated = maybeDirections.orElse(emptyList())
-            .stream()
-            .filter(directionIdVale -> directionIdVale.getValue().getTag().equals(DirectionTag.REQUEST_REASONS_FOR_APPEAL)).findFirst();
+        State currentCaseState = callback.getCaseDetails().getState();
 
+        DirectionTag directionTagToLookFor = stateToDirectionTag(currentCaseState);
+        List<IdValue<Direction>> maybeDirections = getDirections(asylumCase);
```

Comment:
Tiny thing this is no longer maybe the directions as you return an empty list if we don't have them so it should be called directions.

Reply:
true :) will make the change